charts/datadog/templates/agent-priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if or .Values.providers.gke.autopilot .Values.datadog.priorityClassName }}
```

Comment:
IMO it breaks the previous behavior.
.Values.datadog.priorityClassName was use to provide the name of PriorityClass that already exist.

Now it will try to create one with this name.

Reply:
The other approach that I see:
Always create the PriorityClass with a default name. If a user provide a Name it means that the PriorityClass already exist

Reply:
Ah I had misunderstood. I thought this was to specify the name of the PC to be created.
thanks, I will adjust.

Reply:
linked to my other comment

```suggestion
{{- if or .Values.providers.gke.autopilot .Values.agent.createPriorityClass }}
```